config/application.rb

```diff
@@ -15,14 +15,21 @@
 module UFood
   class Application < Rails::Application
     #config.load_defaults 5.1
-    config.generators do |generate|
-      generate.helper false
-      generate.assets false
-      generate.view_specs false
-      generate.helper_specs false
-      generate.routing_specs false
-      generate.controller_specs false
+    config = YAML.load(File.read(File.expand_path('../environment_variables.yml', __FILE__)))
+    config.merge! config.fetch(Rails.env, {})
+    config.each do |key, value|
+      ENV[key] = value.to_s unless value.kind_of? Hash
+
+      config.generators do |generate|
+        generate.helper false
+        generate.assets false
+        generate.view_specs false
+        generate.helper_specs false
+        generate.routing_specs false
+        generate.controller_specs false
+      end
+      config.generators.system_tests = nil
+
```

Comment:
I don't think you need those settings @magnus-thor

Reply:
I am trying to use the ENV for the key, its loading properly in and the key shows up when I goto console and type ENV['GOOGLE_MAP_KEY']